sentry-samples/sentry-samples-spring/build.gradle.kts

```diff
@@ -0,0 +1,40 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+    id(Config.BuildPlugins.springBoot) version Config.springBootVersion
+    id(Config.BuildPlugins.springDependencyManagement) version Config.BuildPlugins.springDependencyManagementVersion
+    kotlin("jvm")
+    kotlin("plugin.spring") version Config.kotlinVersion
+}
+
+group = "io.sentry.sample.spring"
+version = "0.0.1-SNAPSHOT"
+java.sourceCompatibility = JavaVersion.VERSION_1_8
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation("org.springframework.boot:spring-boot-starter-security")
+    implementation("org.springframework.boot:spring-boot-starter-web")
+    implementation("org.springframework.boot:spring-boot-starter")
+    implementation("org.jetbrains.kotlin:kotlin-reflect")
+    implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8")
+    implementation(project(":sentry-spring"))
+    implementation(project(":sentry-logback"))
```

Comment:
is that normal to use always the conjunction of spring + logback? I feel that this would get a little bit more complex than necessary.
these samples are to easier the development and not really for the customer, so mixing spring and logback would make it a little bit harder to test things in insolation.

Reply:
Logback is the default framework that comes with Spring Boot and using these two is I believe the best way to take a full advantage of Sentry capabilities.

Reply:
nice, makes sense then